cli/azd/pkg/azdo/pipeline.go

```diff
@@ -140,7 +146,7 @@ func CreatePipeline(
 func getDefinitionVariables(
 	env *environment.Environment,
 	credentials AzureServicePrincipalCredentials,
-	provisioningProvider provisioning.Options) *map[string]build.BuildDefinitionVariable {
+	provisioningProvider provisioning.Options) (*map[string]build.BuildDefinitionVariable, error) {
```

Comment:
Do we also need to check if `provider.conf.json` is in the infra folder?
Can we get the values we need from there?  (if they are not in the env? )

Reply:
@vhvb1989 we do not store the values directly in the provider.conf.json. we set the values in the Json to variables that gets populated from env vars instead to allow the backend config values to change based on the selected azd env . see the provider.conf.json  sample [here](https://learn.microsoft.com/en-us/azure/developer/azure-developer-cli/use-terraform-for-azd)